### ISSUE TITLE
Conflicts when 2 people join we both get gicked until I delete the focus user

### ISSUE THREAD

**Issue by JoshWho** [igniterealtime/openfire-pade-plugin#175]
What keeps causing focus to kick us from the meeting?

Looks fine when I am in the chat
![7712f5e9-7f1d-47d0-a58b-e570a0815d9c](https://user-images.githubusercontent.com/39392449/103112655-728ffe00-461c-11eb-9f86-736b8ea48405.png)

![34292574-fe30-4424-b6c0-ae342a7ee6c9](https://user-images.githubusercontent.com/39392449/103112663-8471a100-461c-11eb-848c-8b4a771a879a.png)

BUt as soon as another user joins we get kicked

**Comment by JoshWho** [comment]
I forgot to mention by deleting the focus user account allows me to call the other computer. When I restart the plugin I can sometime get it to work with focus user and it shows me in the call like the picture above, but if I leave that room and try to make a new one and invite my bot into it It kicks us out until I delete the focus user again.

**Comment by deleolajide** [comment]
Look for errors in your openfire log files. Something is not working

**Comment by gjaekel** [comment]
In your Video, I see two focus user. This is a show stopper. There's only **one** technical focus account (and **one** JVB account). They are defined at the "Meetings|Setttings" page at the "Security" sections. After changing something here, OpenFire or at least the OFMeet plugin *must* be restarted.

Again, I point out that you have to see exactly *three* JVMs: One for _OpenFire_, one for _JiFoCo_ and one for _JVB2_ - not more, not less.

Again, you may provide a new, clean log of the startup as described before.

**Comment by JoshWho** [comment]
I had a feeling that the focus user is the issue. Like I stated I am new to this but am a fast learner. 

So far what I have learned was, after the plugin ofmeet installs it creates the jvb and focus user account.

When you create a meeting focus should be the admin of the room or at least owner.

I can get the focus user to work correctly after the first initially transaction, meaning creating a conference with success and having more than 3 people using it with no problem.

If I create a new room after the first success the second one fails by kicking everyone out when more than 2 people join.

To me seems like focus loses sync somewhere and goes brain dead. I have to delete the user account focus in order for the chat to work again then reload the plugin to make a successful conf.

I do not believe that this is what it suppose to do so is there anything I am missing on how the focus account works?

As for  @deleolajide  Check the logs,  I wish it was that simple.   Last time the logs didn't help because ofmeet wasn't designed to detect port was in use.  It took several tries and the only thing leading to the port being in use was asking how it worked because there was no instructions at the moment that sated anything about port 8080. Logs did not show me that and only pulled ina error from nginx that was completely useless.

**Comment by deleolajide** [comment]
>Check the logs, I wish it was that simple. Last time the logs didn't help because ofmeet wasn't designed to detect port was in use.

That is true, but sometimes it is reading in-between the log files lines that reveals the info needed especially when they are multiple eyes reading from different perspectives, with different skill-sets and experiences :-)

Port 8080 has changed purpose

**Comment by deleolajide** [comment]
>The active sessions shows 2 focus users but there is only 1 account. Why is focus not leaving the old session when starting a new one?

I believe that is the mystery you have to solve. Confirm that your focus sessions are not anonymous because authentication failure

**Comment by gjaekel** [comment]
> Last time the logs didn't help because ofmeet wasn't designed to detect port was in use.

That's right the other way round: The logs had have an obvious hint that the port was already in use.

**Comment by JoshWho** [comment]
Not sure what but all I did this time was delete the users then try the call, it did the disconnect thing and I killed the extra session., Now it seems to be working fine. SO as long as another session for focus is not running as a ghost it should works good.

I will close this for now unless I see something new that might be a perm fix.

Would be nice for an update of this to have a check for any open sessions of focus and auto kill when not in use or something.
